_topic_maps/_topic_map.yml

```diff
@@ -603,8 +605,8 @@ Topics:
 - Name: Fedora CoreOS (FCOS) image layering
   File: coreos-layering
   Distros: openshift-origin
-- Name: AWS Local Zone tasks
-  File: aws-compute-edge-tasks
+- Name: AWS zone tasks
```

Comment:
I don't think this is a good name for that book. This book includes only tasks for Local or Wavelength zones, if we create this generic approach, imho, could cause confusion with regular zones (Availability Zones), which isn't the case.

I'd suggest to use something like, `AWS edge zone tasks" if you are thinking to group both LZ and WL in the same.